script/tool/lib/src/update_dependency_command.dart

```diff
@@ -181,6 +229,65 @@ ${response.httpResponse.body}
     return PackageResult.success();
   }
 
+  /// Handles all of the updates for [package] when the target dependency is
+  /// an Android dependency.
+  Future<PackageResult> _runForAndroidDependency(
+      RepositoryPackage package) async {
+    if (_targetAndroidDependency == 'gradle') {
+      print('${indentation}Updating examples running on Android...');
+      final Iterable<RepositoryPackage> packageExamples = package.getExamples();
+      bool updateRanForExamples = false;
+      for (final RepositoryPackage example in packageExamples) {
+        if (!example.platformDirectory(FlutterPlatform.android).existsSync()) {
+          continue;
+        }
+
+        updateRanForExamples = true;
+        Directory gradleWrapperPropertiesDirectory =
+            example.platformDirectory(FlutterPlatform.android);
+        if (gradleWrapperPropertiesDirectory
+            .childDirectory('app')
+            .childDirectory('gradle')
+            .existsSync()) {
+          gradleWrapperPropertiesDirectory =
+              gradleWrapperPropertiesDirectory.childDirectory('app');
+        }
+        final File gradleWrapperPropertiesFile =
+            gradleWrapperPropertiesDirectory
+                .childDirectory('gradle')
+                .childDirectory('wrapper')
+                .childFile('gradle-wrapper.properties');
+
+        final String gradleWrapperPropertiesContents =
+            gradleWrapperPropertiesFile.readAsStringSync();
+        final RegExp validGradleDistributionUrl =
+            RegExp(r'^\s*distributionUrl\s*=\s*.*\.zip', multiLine: true);
+        if (!validGradleDistributionUrl
+            .hasMatch(gradleWrapperPropertiesContents)) {
+          return PackageResult.fail(<String>[
+            'Unable to find a "distributionUrl" entry to update for ${package.displayName}.'
+          ]);
+        }
+
+        print('${indentation}Updating to "$_targetVersion"');
```

Comment:
Since this is in a loop, we should say which example:
`print('${indentation}Updating ${getRelativePosixPath(example.directory,
          from: package.directory)} to "$_targetVersion"');`